packages/backend-core/src/environment.ts

```diff
@@ -202,6 +202,21 @@ const environment = {
   DISABLE_SCIM_CALLS: process.env.DISABLE_SCIM_CALLS,
 }
 
+type EnvironmentKey = keyof typeof environment
+export const SECRETS: EnvironmentKey[] = [
+  "API_ENCRYPTION_KEY",
```

Comment:
There are some environment variables which are specific to services, like `OPENAI_API_KEY`, `BB_ADMIN_USER_PASSWORD`, `SENDGRID_API_KEY`, `AWS_SESSION_TOKEN` which are from the `packages/server/src/environment.ts` and `packages/worker/src/environment.ts` - these may not be in the backend-core environment list, although all of the ones I've listed are common.

One thing I've thought about doing is getting rid of the worker/server environment implementations and keeping one full list in backend-core, which helps a bit with managing the secret list as well.